### ISSUE TITLE
Vercel CLI 33.6.0 deploy failing with error: Invalid request: `projectSettings.nodeVersion` should be equal to one of the allowed values "20.x, 18.x, 16.x".

### ISSUE THREAD

**Issue by Zhaph** [zeit/now#11285]
Our CI/CD deployments started failing this morning when they upgraded the version of the CLI used from 33.5.5 to 33.6.0.

Attempting to pull the repo and deploy using:

```
- task: bash@3
  displayName: 'Deploy Front End to Vercel Production'
  inputs:
    targetType: 'inline'
    script: |
      npm install -g vercel
      vercel --version
      vercel deploy --prod --token ${{ parameters.VercelToken }}
  env:
    VERCEL_ORG_ID: ${{ parameters.VercelTeamId }}
    VERCEL_PROJECT_ID: ${{ parameters.VercelProjectId }}
```

Now results in:

> Error: Invalid request: `projectSettings.nodeVersion` should be equal to one of the allowed values "20.x, 18.x, 16.x".

Previously this worked as expected and deployed successfully.

The Project referenced in the `VERCEL_PROJECT_ID` environment variable has it's node version set to `18.x`.

**Comment by lanlin** [comment]
same problem

**Comment by swarnava** [comment]
Hi, Thanks for reporting. We will fix this in next version of CLI v33.6.1: https://github.com/vercel/vercel/pull/11277. In the meantime, you can change `engines` in `package.json` to `16.x`, `18.x` or `20.x`.

Thank you!

**Comment by swarnava** [comment]
Hi,

We have fixed the issue which will be available in next version of CLI release v33.6.1. You can revert the CLI installation command to original command (npm install -global vercel@latest) to ensure you are always on deploying using latest version of CLI.

Feel free to reach out to us if you have any questions.
